apps/opencs/model/world/cellcoordinates.hpp

```diff
@@ -7,6 +7,8 @@
 
 #include <QMetaType>
 
+#include <osg/PositionAttitudeTransform>
```

Comment:
Is this include needed? I don't see any `osg::PositionAttitudeTransform`s anywhere.